tests/data_asset/test_parameter_substitution.py

```diff
@@ -1,384 +0,0 @@
-"""
-Test the expectation decorator's ability to substitute parameters
-at evaluation time, and store parameters in expectation_suite
-"""
```

Comment:
We don't care about this decorator in V1 and already have other tests for evaluation parameter substitution with expectations - found it simpler to just delete than debug